internal/postgres/convert.go

```diff
@@ -0,0 +1,369 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package postgres
+
+import (
+	"fmt"
+	"harbourbridge/internal"
+	"harbourbridge/spanner/ddl"
+	"reflect"
+	"strings"
+	"time"
+
+	nodes "github.com/lfittl/pg_query_go/nodes"
+)
+
+// Conv contains all schema and data conversion state.
+type Conv struct {
+	mode           bool                       // False is schema mode, true is data mode.
```

Comment:
Could this be an enum for better readability?

Reply:
Done.